grr/server/grr_response_server/gui/ui/index.html

```diff
@@ -19,4 +19,4 @@
   </script>
   <app-root></app-root>
 </body>
-</html>
+</html>
```

Comment:
Please add a trailing newline to all text files. The same applies to proxy.conf.json.